front/src/App.tsx

```diff
@@ -52,10 +52,20 @@ export function App() {
   return (
     <DefaultLayout>
       <Routes>
+        <Route
+          path="auth/*"
+          element={
+            <RequireOnboarding>
```

Comment:
I did not understand this naming at first.
Maybe `RequireNotOnboarded` or `RequireOnboardingUncompleted` ?